src/main/java/org/folio/consortia/service/impl/UserTenantServiceImpl.java

```diff
@@ -230,7 +234,7 @@ private void createOrUpdateShadowUser(UUID userId, User shadowUser, UserTenant u
       if (Objects.nonNull(user.getActive())) {
         activateUser(user);
       } else {
-        createActiveUser(shadowUser);
+        createActiveUserWithPermission(shadowUser);
```

Comment:
lets rename this method to createActiveUserWithPermissions

Reply:
fixed